packages/xrpl/webpack.config.js

```diff
@@ -64,8 +65,7 @@ function getDefaultConfiguration() {
       symlinks: false,
       fallback: {
         buffer: require.resolve('buffer/'),
-        stream: require.resolve('stream-browserify'),
-        crypto: require.resolve('crypto-browserify'),
+        stream: require.resolve('stream-browserify')
```

Comment:
@ckniffen 

It errored when I tried to remove this. Maybe I was confused by the diff, and the removal of the trailing `,` was all happened, and you didn't actually remove the stream fallback in 3.0 branch? 

https://github.com/XRPLF/xrpl.js/actions/runs/5735287942/job/15542792906#step:7:29

Reply:
Confusion confirmed:
https://github.com/XRPLF/xrpl.js/blob/6bdcf629364061a683b148f042547be19e47473f/packages/xrpl/webpack.config.js#L65C1-L69C9

Reply:
For context `stream-browserify` is used by https and crypto polyfills. Removing the crypto polyfills was the last part needing to be removed.

We also need to update UNIQUE_SETUPS.md to remove polyfills.  We are now down to just `buffer` and `process`.  I have some rough POCs that remove buffer but that I will relegate to a future PR.

Reply:
The webpack.config in the test folder should also be updated to remove several of the polyfills.